File: RainbowSpectrum/preprocessing/spa_converter.py
import numpy as np
import os
from LoadSpectrum import read_spa

raw_path=r"G:\My Drive\PhD\Doktorat\FTIR analysis\Spektri master\.SPA SIROVO DEKI\Lazar 1.spa"
os_path=raw_path.replace('\\', '/')
file_path = os.path.abspath(os_path)




spectra, wavelength, title = read_spa(file_path)

print(wavelength)

File: SpectralTool/rainbow_spectrum_script/spa_converter.R
#setup work directory
setwd("G:/My Drive/PhD/Doktorat/FTIR analysis/Spektri master/.SPA SIROVO DEKI")

# load OpenSpecy
library(OpenSpecy)

# read . spa file
spektar = read_spa("Lazar 16.spa")

# plot the OpenSpeccy object
plotly_spec(spektar)

# convert object to dataframe
df = as.data.frame(spektar)
View(df)

# delete first row in dataframe
df=df[- 1, ]
View(df)

